server/templates/index.html

```diff
@@ -7,7 +7,7 @@
   <meta name="viewport"
     content="width=device-width, initial-scale=1, user-scalable=0, maximum-scale=1, minimum-scale=1"
   >
-  <link rel="icon" type="image/png" href="{{ url_for('static', filename='favicon.png') }}" />
+  <link rel="icon" type="image/png" href="{{ url_for('static', filename='favicon-32x32.png') }}" />
```

Comment:
Include support for all favicon types:

```suggestion
  <link rel="apple-touch-icon" sizes="180x180" href={{ url_for('static', filename='apple-touch-icon.png') }}">
  <link rel="icon" type="image/png" sizes="32x32" href="{{ url_for('static', filename='favicon-32x32.png') }}">
  <link rel="icon" type="image/png" sizes="16x16" href="{{ url_for('static', filename='favicon-16x16.png') }}">
  <link rel="manifest" href="{{ url_for('static', filename='site.webmanifest') }}">
  <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico') }}">
  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#022069">
  <meta name="msapplication-TileColor" content="#2b5797">
  <meta name="theme-color" content="#022069">
```

[favicon_package_v0.16.zip](https://github.com/CodeForAfrica/web-tools/files/15496918/favicon_package_v0.16.zip)